content/en/docs/concepts/configuration/secret.md

```diff
@@ -24,6 +24,16 @@ a password, a token, or a key. Such information might otherwise be put in a
 Pod specification or in an image. Users can create Secrets and the system
 also creates some Secrets.
 
+{{< caution >}}
+Kubernetes Secrets are, by default, stored as unencrypted base64-encoded
+strings. By default they can be retrieved - as plain text - by anyone with API
+access, or anyone with access to Kubernetes' underlying data store, etcd. In
+order to safely use Secrets, we recommend you (at a minimum):
+
+1. [Enable Encryption at Rest](https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/) for Secrets.
```

Comment:
Hi @adamhjk . Noted a couple of style nits, otherwise, looks OK.
nit: avoid "we".  Who recommends? You could write something like,
`To safely use Secrets, it is recommended that you ...`
nit: Use intra-site links:
```
[Enable Encryption at Rest](/docs/tasks/administer-cluster/encrypt-data/)
[Enable RBAC rules that restrict reading and writing the Secret](/docs/reference/access-authn-authz/authorization/)
```

Reply:
I agree, other than that the commit looks great.

Reply:
```suggestion
1. [Enable Encryption at Rest](/docs/tasks/administer-cluster/encrypt-data/) for Secrets.
```